src/com/fedorowiat/menu/LoginMenu.java

```diff
@@ -15,13 +17,22 @@ public LoginMenu() {
         this.input = new Scanner(System.in);
     }
 
-    public void showLoginMenu() {
+    public void showLoginMenu(DataBase dataBase) {
         System.out.print("\n\nPodaj login: ");
-        String login = input.nextLine();
-
+        var login = input.nextLine();
         System.out.print("Podaj hasło: ");
-        String password = input.nextLine();
+        var password = input.nextLine();
         System.out.println("\n\n");
+            if(dataBase.checkLoginData(login, password)){
+                if (dataBase.getAccessLevel(login, password).equals("false")){
+                    userMenu.showUserMenu(login);
+                } else if (dataBase.getAccessLevel(login, password).equals("true")) {
```

Comment:
moze zamiast true i false to daj ADMIN i USER, bo tak to mozesz to sparsowac do booleana i zobic tak: 
`if (Boolean.parseBoolean(dataBase.getAccessLevel(login, password)))`